testcafe/tests/pages/drive/drive-model-private.js

```diff
@@ -1,13 +1,40 @@
 import { t } from 'testcafe'
 import {
+  getElementWithTestId,
   getPageUrl,
   isExistingAndVisibile,
   overwriteCopyCommand,
   getLastExecutedCommand
 } from '../../helpers/utils'
 import DrivePage from './drive-model'
 
-export default class PrivateDrivePage extends DrivePage {
+import Modal from './drive-modal-model'
+const moveModale = new Modal()
+
+export default class privateDrivePage extends DrivePage {
+  constructor() {
+    super()
+    this.elementActionMenuByRowIndex = value => {
```

Comment:
Shouldn't this be a class method?

Reply:
well `this.elementActionMenuByRowIndex` is a selector (with a parameter) and I already have a few of them written that way (ie: https://github.com/cozy/cozy-drive/blob/master/testcafe/tests/pages/photos/photos-model.js#L39)

Reply:
Ok, hadn't noticed it before. Maybe an improvement for another PR.